src/Configuration/Provisioning/data/hooks.js

```diff
@@ -78,11 +79,24 @@ export default function useProvisioningContext() {
 
   const hydrateCatalogQueryData = useCallback(async () => {
     const { data } = await LmsApiService.fetchEnterpriseCatalogQueries();
+    const camelCasedData = camelCaseObject(data.results);
+    const predeterminedCatalogQueries = camelCasedData.filter(({ title }) => !title.search(/\[DO NOT ALTER\]\[LEARNER CREDIT\]/));
```

Comment:
Nit: if we switch to using `title.indexOf()` we can be more DRY and pull out the prefix into a const (or even in `../../data/constants`???) and used everywhere.  E.g.:
```suggestion
    const learnerCreditCatalogPrefix = '[DO NOT ALTER][LEARNER CREDIT]'
    const predeterminedCatalogQueries = camelCasedData.filter(
      ({ title }) => title.indexOf(learnerCreditCatalogPrefix) != 0
    );
```

and below:

```python
      if (catalogQueryObject.title.split(learnerCreditCatalogPrefix + ' ')[1] === catalogQuery.catalogQueryTitle.split(' account')[0]) {
```

Reply:
Implemented the first comment solution as it removes the need to use regex's 👍🏽 . Once we brought in the ids through the configuration, it was a solution that allowed it to work on local stage and prod environments removing the need to filter twice.